parser/spdx/spdx_test.go

```diff
@@ -0,0 +1,81 @@
+package spdx
+
+import "testing"
+
+func TestSPDXParser(t *testing.T) {
+	p := `SPDXVersion: SPDX-2.2
```

Comment:
nit: This test string seems excessively long if we're only checking one field in it.

Reply:
thank you so much @jalseth, done.